### PR TITLE
Add Prettier automation

### PR DESCRIPTION
**Pull request by LaChope**
Prettier will now format files before commit automatically

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (blcham) [inline .husky/pre-commit:1]: I don't get it, there is no husky.sh script so do not understand why calling it ... so i assume husky install do that ...

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+. "$(dirname "$0")/_/husky.sh"
+
+npm run prettier && git add -A .
```
Comment (blcham) [inline .husky/pre-commit:4]: This is little dangerous from my point of view. As you might want to commit only part of the file but you will get full file after it ... 

I will add some notes to the ticket (not pull request)

**Review by blcham — APPROVED** [review]
Approve, but don't understand the workflow, for next time I suggest discussing the workflow first in the ticket (I know that it is hard for u to decide when to do it and when not :).

File: .husky/pre-commit (hunk 1 above)
Comment (LaChope) [reply .husky/pre-commit:1]: Yes exactly, husky.sh is created with husky install

**Review by blcham — COMMENTED** [review]
Submitting again as i saw some pending comments.

**Comment by blcham** [comment]
@LaChope I am not sure what to do with the pull request, there is 72 files changed ... i think we discussed it yet, but don't remember the resolution

**Comment by LaChope** [comment]
Closing in favor of #142

**Pull request closed without merging**